### PR TITLE
updated database and minor improvements

### PR DESCRIPTION
**Pull request by PauliusKibilda**

### CODE REVIEW

File: FoodWaste/Controllers/ProductsController.cs [hunk 1]
```diff
@@ -264,84 +223,52 @@ public async Task<IActionResult> Edit(int? id)
...
-            return View();
+
+            return _context.Product.Any(e => e.Id == id);
+
         }
 
         private int GetCurrentUserId()
         {
-            
+
```
Comment (VaidasCib) [inline FoodWaste/Controllers/ProductsController.cs:256]: fix spacing

File: FoodWaste/Controllers/ProductsController.cs [hunk 2]
```diff
@@ -264,84 +223,52 @@ public async Task<IActionResult> Edit(int? id)
...
-            {
-                return false;
-            }
-        }
-        public async Task<IActionResult> NotFound(int? id) 
-        {
-            return View();
+
+            return _context.Product.Any(e => e.Id == id);
+
```
Comment (VaidasCib) [inline FoodWaste/Controllers/ProductsController.cs:251]: spacing again

File: FoodWaste/Controllers/DataBaseOperations.cs [hunk 3]
```diff
@@ -53,14 +53,14 @@ public static async Task<List<Product>> UpdateProducts()
                     {
                         string apiResponse = await response.Content.ReadAsStringAsync();
                         var productResponse = JsonConvert.DeserializeObject<ObservableCollection<Product>>(apiResponse);
-                        foreach(var p in productResponse)
+                        foreach (var p in productResponse)
                         {
                             products.Add(p);
```
Comment (PauliusKibilda) [inline FoodWaste/Controllers/DataBaseOperations.cs:58]: addrange

File: FoodWaste/Controllers/ProductsController.cs [hunk 4]
```diff
@@ -76,51 +72,43 @@ public async Task<IActionResult> Reserve(int? id)
...
                 return NotFound();
             }
-            if (! await IsCurrentUserRestaurant())
+            if (IsCurrentUserRestaurant())
             {
-                //var product = await _context.Product
-                //    .FirstOrDefaultAsync(m => m.Id == id);
-                try
+                var product = await _context.Product
+                .FirstOrDefaultAsync(m => m.Id == id); ;
```
Comment (PauliusKibilda) [inline FoodWaste/Controllers/ProductsController.cs:78]: FindAsync

Comment (PauliusKibilda) [reply FoodWaste/Controllers/ProductsController.cs:78]: additional ;

**Pull request merged**